protocol/x/clob/memclob/memclob.go

```diff
@@ -495,14 +495,29 @@ func (m *MemClobPriceTimePriority) PlaceOrder(
 	}
 
 	if m.generateOffchainUpdates {
-		// If this is a replacement order, then ensure we send the appropriate replacement message.
+		// If existing order ID is found and the price of the existing order is different from the new order,
+		// create an order removal message and then an order place message.
+		// If the price of the existing order is the same as the new order, don't send any messages here because
```

Comment:
Not sure I follow why no message needs to be sent if price is the same.
What if the order isn't matched?

Reply:
I think that will get handled by the Order Update message sent at the end of this function